src/resources/questions/common.ts

```diff
@@ -76,48 +76,56 @@ export function ensureCatchAllResponse(responses: any[]): void {
   if (!hasCatchAllRule(responses)) responses.push(makeCatchAllResponse());
 }
 
+/*
+ * Same as wrapInlinesWithParagraphs, but if there are no children, returns an empty paragraph.
+ */
 export function ensureParagraphs(children: any[]) {
-  // if all children are text|inline elements: wrap all in single p
-  if (children.every((c: any) => c.text !== undefined || isInlineTag(c.type))) {
-    const withEmptyText = children.length === 0 ? [{ text: ' ' }] : children;
-    return [{ type: 'p', children: withEmptyText }];
-  }
-
-  return children;
+  const result = wrapInlinesWithParagraphs(children);
+  return result.length === 0
+    ? [{ type: 'p', children: [{ text: ' ' }] }] // Add an empty p if no children at all.
```

Comment:
Is it possible to simply use an empty string`''` instead of a single space here? That would allow the editor to indicate as it currently does "Type here or use + to get started..." and indicate there is an empty editable area to the user

Reply:
Yes, we could.

The downside is it introduces more potential differences between the old & new migration output since the old ensureParagraphs had the space.

I don't have a strong opinion one way or the other.